spec/views/devise/mailer/confirmation_instructions.html.slim_spec.rb

```diff
@@ -11,10 +11,7 @@
   end
 
   it 'includes the support link' do
-    assign(:resource, build_stubbed(:user, confirmed_at: Time.zone.now))
-    render
-
-    expect(rendered).to have_link(Figaro.env.support_url, href: Figaro.env.support_url)
+    to include Figaro.env.support_url
```

Comment:
This seems to be a copy and paste error. Looks like the `expect(rendered)` got dropped off. Also, was there a reason you removed the first part of this test?